tests/test_pdf_cli.py

```
@@ -0,0 +1,45 @@
+fitz = pytest.importorskip("fitz")
+
+
+def test_pdf_cli_creates_txt_and_optional_epub(tmp_path):
+    pdf_path = tmp_path / "cli_test.pdf"
+    doc = fitz.open()
+    page = doc.new_page()
+    page.insert_text((72, 72), "CLI test\nPDF content")
+    doc.save(str(pdf_path))
+
+    # run CLI
+    subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)